src/pages/WorkloadDetails/WorkloadDescription.tsx

```diff
@@ -0,0 +1,171 @@
+  render() {
+    const workload = this.props.workload;
+    const apps: string[] = [];
+    const services: string[] = [];
+
+    if (workload) {
+      if (workload.labels[serverConfig.istioLabels.appLabelName]) {
+        apps.push(workload.labels[serverConfig.istioLabels.appLabelName]);
+      }
+      workload.services.forEach(s => services.push(s.name));
+    }
+
+    const isTemplateLabels =
+      workload &&
+      ['Deployment', 'ReplicaSet', 'ReplicationController', 'DeploymentConfig', 'StatefulSet'].indexOf(workload.type) >=
+        0;
+    const runtimes = (workload?.runtimes || []).map(r => r.name).filter(name => name !== '');
+
+    const workloadProperties = workload ? (
+      <>
+        <div key="properties-list" className={resourceListStyle}>
+          <ul style={{ listStyleType: 'none' }}>
+            {workload.istioInjectionAnnotation !== undefined && (
+              <li>
+                <span>Istio Injection</span>
+                {String(workload.istioInjectionAnnotation)}
+              </li>
+            )}
+            <li>
+              <span>Type</span>
+              {workload.type ? workload.type : 'N/A'}
+            </li>
+            <li>
+              <span>Created</span>
+              <div style={{ display: 'inline-block' }}>
+                <LocalTime time={workload.createdAt} />
+              </div>
+            </li>
+            <li>
+              <span>Version</span>
+              {workload.resourceVersion}
+            </li>
+            {workload.additionalDetails.map((additionalItem, idx) => {
+              return (
+                <li key={'additional-details-' + idx} id={'additional-details-' + idx}>
+                  <span>{additionalItem.title}</span>
+                  {additionalItem.icon && renderAPILogo(additionalItem.icon, undefined, idx)}
+                  <TextOrLink text={additionalItem.value} urlTruncate={64} />
+                </li>
+              );
+            })}
+            {runtimes.length > 0 && (
+              <li id="runtimes">
+                <span>Runtimes</span>
+                <div style={{ display: 'inline-block' }}>
+                  {runtimes
+                    .map((rt, idx) => renderRuntimeLogo(rt, idx))
+                    .reduce(
+                      (list: JSX.Element[], elem) =>
+                        list.length > 0 ? [...list, <span key="sep"> | </span>, elem] : [elem],
+                      []
+                    )}
+                </div>
+              </li>
+            )}
+          </ul>
+        </div>
+      </>
+    ) : undefined;
+
+    return workload ? (
+      <Card id={'WorkloadDescriptionCard'}>
+        <CardHeader>
+          <Title headingLevel="h5" size="lg">
+            <div key="service-icon" className={iconStyle}>
+              <Tooltip position={TooltipPosition.top} content={<>Workload</>}>
+                <Badge className={'virtualitem_badge_definition'}>W</Badge>
+              </Tooltip>
```

Comment:
Please replace this with the new PFBadge:

`<PFBadge badge={PFBadges.Workload} position={TooltipPosition.top} />`